linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/oval/shared.xml

```diff
@@ -0,0 +1,33 @@
+<def-group>
+  <definition class="compliance" id="rsyslog_remote_tls" version="1">
+    <metadata>
+      <title>Check that rsyslog is configured to use TLS for remote logging</title>
+      <affected family="unix">
+        <platform>multi_platform_fedora</platform>
+        <platform>Red Hat Enterprise Linux 8</platform>
+        <platform>Oracle Linux 8</platform>
+      </affected>
+      <description>Check that all needed TLS-related options are present</description>
+    </metadata>
+    <criteria comment="Check that rsyslog is configured to use TLS for remote logging" operator="AND">
+      <criterion comment="Check that all needed TLS-related options are present" test_ref="test_rsyslog_remote_tls" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test id="test_rsyslog_remote_tls"
+    comment="tests the omfwd action configuration"
+    check_existence="at_least_one_exists" check="all" version="1">
+    <ind:object object_ref="obj_rsyslog_remote_tls" />
+    <ind:state state_ref="state_rsyslog_remote_tls" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="obj_rsyslog_remote_tls" version="1">
+    <ind:filepath operation="pattern match">^/etc/rsyslog\.(conf|d/.+\.conf)$</ind:filepath>
+    <ind:pattern operation="pattern match">^\s*action\(type="omfwd"(.+?)\)</ind:pattern>
```

Comment:
@adelton This works only if the whole command is on a single line. From a quick look into rsyslog conf, it seems it's possible to have newlines and other whitespace characters inside the parentheses. Could you take a look at it?

Reply:
you can try `<behaviors singleline="true"/>`

Reply:
Added as 843c7f1978ed43324fa621888afdc2563b06e124.